src/ViewInstance.js

```diff
@@ -203,13 +196,37 @@ class ViewInstance extends React.Component {
     });
   };
 
+  handleViewSource = (e, instance) => {
```

Comment:
Please make it a regular method (not an arrow function) as it does not need to be bound.